subsys/net/lib/lwm2m/Kconfig

```diff
@@ -122,6 +122,7 @@ config LWM2M_FIRMWARE_UPDATE_PULL_SUPPORT
 	bool "Firmware Update object pull support"
 	default y
 	depends on LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT
+	select HTTP_PARSER_URL_ONLY
```

Comment:
What about my comment here, is the HTTP_PARSER option selected by some other way? What I mean by this is if you select HTTP_PARSER_URL_ONLY here, it will not select HTTP_PARSER automatically (at least that is how Kconfig worked before) i.e., recursive "select" is not implemented. But it might be that the HTTP_PARSER is selected automatically by some other mean in which case my concern is moot.